test/functionalTest/cases/0725_multiservice_ignore/service_ignore_without_multiservice.test

```diff
@@ -101,10 +102,11 @@ Date: REGEX(.*)
 =====================================================
 1
 0
+ftest
 --TEARDOWN--
```

Comment:
Let's keep it, it doesn't hurt. However, I have provided a comment explaining where this come from.

Fixed in 53882e4